tests/async_mock.py

```diff
@@ -0,0 +1,43 @@
+"""Mock utilities that are async aware."""
+
+from unittest.mock import *  # noqa: F401, F403
+
+
+class _IntSentinelObject(int):
```

Comment:
Hmm, I don't really like that we need to port this over from zigpy, but it's fine..